docs/contracts/overview.md

```diff
@@ -30,8 +33,8 @@ two for security deposits:
 * The Seller's deposit  
 * The Buyer's deposit  
 
-Supported **currencies** are currently: ETH and $BOSON tokens.
-> Note: Functions dealing with funds have suffices such as ETHETH or ETHTKN to 
+Supported **currencies** are currently: ETH, $BOSON tokens and DAI.
```

Comment:
s/tokens//

Reply:
I don't understand what you mean here. Do you mean remove the worked tokens?

Reply:
sorry tokens for token ? We only have one BOSON token right now ?

Reply:
Changed. Will be included in next commit.